aquality-automation/aquality-tracking-api
# Feature/public api

# PR Details

Added public API methods to simplify importing test results via API

## Related Issue

  - Provide public API -> [View Issue](https://github.com/aquality-automation/aquality-tracking/issues/56)

## How Has This Been Tested

Use Swagger for Public section
You should be able

- Create/Update Test Suite
- Create/Update Test and add it to suite
- Start/Finish test run for suite
- Start/Finish test result for test and test run

## Checklist
